qutebrowser/config/configdata.yml

```diff
@@ -812,7 +812,26 @@ completion.timestamp_format:
   default: '%Y-%m-%d'
   desc: Format of timestamps (e.g. for the history completion).
 
+completion.web_history.exclude:
+  type:
+    name: List
+    valtype: UrlPattern
+    none_ok: true
+  default: []
+  restart: true
+  desc: >-
+    A list of patterns which should not be shown in the history.
+
+    This only affects the completion. Matching URLs are still saved in the
+    history (and visible on the qute://history page), but hidden.
```

Comment:
"...saved... and visible... but hidden."

It would be good to finish the sentence, e.g. by adding explicitly where they are hidden.